morse_module/morse.c

```diff
@@ -146,6 +170,16 @@ static inline void setup_mcu() {
 	gpio_pull_down(FREQ_INC);
 	gpio_pull_down(TX);
 
+	// Setting up the addr bits
+	int pins[4] = {ADDR0, ADDR1, ADDR2, ADDR3};
```

Comment:
you don't need to specify the array length here
and you can constify this

Reply:
consted